src/js/clickable-component.js

```diff
@@ -130,6 +130,30 @@ class ClickableComponent extends Component {
   }
 
   /**
+   * Enable the component element
+   *
+   * @return {Component}
+   * @method enable
+   */
+  enable() {
+    this.removeClass('vjs-disabled');
+    this.el_.setAttribute('aria-disabled', 'false');
```

Comment:
not necessary for this, but we're doing a lot of attribute setting, we probably should make a helper for components so you could just do `this.setAttribute('aria-disabled', 'false');`